src/libs/constants.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Number of iterations to execute when hashing the password
+ *
+ * NIST recommends at least 10,000 iterations (https://pages.nist.gov/800-63-3/sp800-63b.html#sec5),
+ */
+export const HASH_ITERATIONS = 10000;
+
+/**
+ * Size of the key to hash the password
+ */
+export const HASH_KEY_SIZE = 256;
```

Comment:
We should add an important note on the docstring here.
That if these values change all passwords on the database will become invalid

Reply:
Good point. Added a warning on [8e2ccda](https://github.com/HathorNetwork/hathor-atomic-swap-service/pull/1/commits/8e2ccda02dccb5409ec1d54365c1092b0de482d3)